src/Plugin/DebugDataItem/Tunnistamo.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\helfi_tunnistamo\Plugin\DebugDataItem;
+
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\helfi_api_base\DebugDataItemPluginBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Plugin implementation of the debug_data_item.
+ *
+ * @DebugDataItem(
+ *   id = "tunnistamo",
+ *   label = @Translation("Tunnistamo"),
+ *   description = @Translation("Tunnistamo")
+ * )
+ */
+class Tunnistamo extends DebugDataItemPluginBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
```

Comment:
This can be removed since nothing is added here. You might have to remove `implements ContainerFactoryPluginInterface` as well.